lib/rock/packaging/packageinfoask.rb

```diff
@@ -0,0 +1,177 @@
+
+module Autoproj
+    module Packaging
+        class PackageInfoAsk
+
+            class << self
+                alias :class_new :new
+
+                def new(which, options)
```

Comment:
Cannot return a object of a different class from initialize, just initialize it.